tslint.json

```diff
@@ -11,7 +11,8 @@
   },
   "linterOptions": {
     "exclude": [
-      "src/*.spec.tsx"
+      "src/*.spec.tsx",
+      "src/stories/*"
```

Comment:
Bunu neden exclude ettin?

Reply:
Lambdas are forbidden in JSX attributes due to their rendering performance impact. 

Bu ve bunun gibi hatalarla karşılaşıyoruz storylerimizde. Storyler kütüphaneyi kullananın kendi lint ayarlarına bağlı olcağı yeri temsil ettiği için bizim componentler için yaptığımız lint ayarlarımız ile storyler uyuşmayabiliyor. Çözüm olarak storyleri kaldırabiliriz lintimizden ya da daha iyisi tslint ayarlarımızı güncelleriz.

Reply:
Lambda kullanma konusunda dikkat etmemiz gerekiyor.
`onClick={(e) => e.preventDefault()}` şeklinde yapmak yerine bunu bir `handleClick` üzerine almak her zaman alışkanlık edinmemiz gereken bir durum. Bence bunu exclude etmememiz lazım

Reply:
Şu an zaten öyle sadece bazı storylerde kullanım gösterirken lambda kullanıldı.

Reply:
Bu hataları gidermek çok zamanını alacak şeyler değil gibi. halledebilir misin?

Reply:
Değiştirdim 👍